taskfiles/rp.yml

```diff
@@ -61,7 +61,7 @@ tasks:
     cmds:
     - |
       PATH={{.LLVM_INSTALL_PATH}}/bin:$PATH
-      ninja -C '{{.BUILD_DIR}}' -j'{{div .TOTAL_PHYSICAL_MEMORY .GB_PER_BUILD_CORE}}'
+      ninja {{.TARGETS}} -C '{{.BUILD_DIR}}' -j'{{div .TOTAL_PHYSICAL_MEMORY .GB_PER_BUILD_CORE}}'
```

Comment:
can you specify in the commit, how to call this? 

`TARGETS=redpanda task rp:build` ?

Reply:
both should (as environment variable or as task variable).

An alternative is to use `{{.CLI_ARGS}}` instead of `{{.TARGETS}}`. `CLI_ARGS` is a built-in variable that passes the arguments given in the command line, after the `--` end-of-arguments delimiter. So it would look like:

```bash
task rp:build -- target_foo target_bar etc
```

Reply:
i think that's fine, i just wanted to know how to use it.